.github/workflows/initialstate-kpis.yml

```diff
@@ -20,4 +19,4 @@ jobs:
           ISS_ACCESS_KEY:  ${{ secrets.ISS_ACCESS_KEY }}
       run: |
         pip install -r ./iss/requirements.txt
-        python ./iss/maintainer_stats.py --debug --small-terminal -n "Tektronix Open Source :ambulance: Health Monitor :ambulance:" -b CW37LU8VCTQH -r vscode-tsplang
+        python ./iss/maintainer_stats.py --debug --small-terminal -n "Tektronix Open Source :ambulance: Health Monitor :ambulance:" -k CW37LU8VCTQH -r vscode-tsplang -b master
```

Comment:
Not sure what branch you had in mind... dev?

Reply:
`dev` and `master` if possible. See my changes to [syphon#21](https://github.com/tektronix/syphon/pull/21) for implementation details.